### ISSUE TITLE
Chore: "We can't find the internet ... Attempting to reconnect"

### ISSUE THREAD

**Issue by nelsonic** [dwyl/image-classifier#11]
Pretty much every time I attempt to view the app on Fly: https://imgai.fly.dev I get the `LiveView` connection Flash:
<img width="1092" alt="image" src="https://github.com/dwyl/image-classifier/assets/194400/89481bbf-a923-4683-8a85-da3afd6cf295">

<img width="811" alt="phoenix-liveview-we-cant-find-the-internet" src="https://github.com/dwyl/image-classifier/assets/194400/f24b18ef-851c-461b-a0a7-eb771e741fec">

# Todo

+ [ ] Can we _track_ how many people are seeing this error? 💭 (i.e. keep a log of connections and disconnections ...)
  + [ ] Please time-box `T25m` of Googling and comment with your findings ... 🔍 
+ [ ] Figure out if we can `pause` the instance after a specific amount of time e.g. `T10m` 

If `people` are seeing this error we need to revisit what we used to do on Heroku to wake-up a dormant app. 😴 
See: https://github.com/dwyl/ping 
Basically we would trigger the app waking up when someone was viewing the `README.md` by including a `GIF` in it ...
Related to: https://github.com/dwyl/image-classifier/issues/6 increasing the machine size will incur more cost. 💸 
if we run the app for a _minimum_ of `T10m` each time it will only cost us `$0.01` (one cent) per run with a cap at `$41/month`.

> [!NOTE]
> The _reason_ we are investigating this for this stand-alone app is that our `mvp` uses `LiveView` 📱 
> and we need know when `people` are seeing the "We can't find the internet" Error ... 😞 
> We could shift our findings/write-up elsewhere in terms of documentation e.g: 
https://github.com/dwyl/phoenix-liveview-counter-tutorial
> But for now we just need to capture what we learn as comments in this thread.

**Comment by ndrean** [comment]
So if you inspect, you have 2 phx-disconnected. If you remove the 1st line `<.flash_group flash={@flash} />` from "page_live.html.heex",  it should be ok.
